cfpb/consumerfinance.gov
# TCCP: Dataset format updates

This PR adds 4 new fields to the TCCP CardSurveyData model: `advance_apr_no_score`, `intro_apr_no_score`, `purchase_apr_no_score`, and `transfer_apr_no_score`. These fields hold information on APRs available to consumers with no credit score. These fields are optional in the dataset and are not used in the "Explore credit cards" tool anywhere.

This PR also renames the `top_25_institution` column to `issued_by_top_25_institution` for clarity.

## How to test this PR

There should be no user-visible changes.

If you run a local server and load the forthcoming dataset (ask me), the "Small institution" checkbox should continue to work properly.

## Checklist
